src/main/java/net/hypixel/nerdbot/listener/ActivityListener.java

```diff
@@ -198,6 +199,33 @@ public void onReactionReceived(MessageReactionAddEvent event) {
             return; // Ignore Empty User
         }
 
+        // Here is logic for handling pinning of messages within Threads by the owner. This is done by only the :pushpin: Emoji.
+        if (event.getReaction().getEmoji().getName().equals("\uD83D\uDCCC")) {
+            Emoji pushpin = Emoji.fromUnicode("\uD83D\uDCCC");
+
+            if (!(event.getGuildChannel() instanceof ThreadChannel)){

```

Comment:
Could probably use pattern matching on ThreadChannel and use it further down
Also, formatting pls

Reply:
Would actually also probably swap this and line 203